test/Android/Breakpoint/TestScript.xml

```diff
@@ -111,7 +111,7 @@
     <Event name="IDebugThreadCreateEvent2" expected="False" />
     <Event name="IDebugOutputStringEvent2" expected="False" />
     <Event name="IDebugBreakpointBoundEvent2" expected="True">
-      <BoundBreakpoint>24,0 to 24,0 in [Unknown]!break_test_1()</BoundBreakpoint>
+      <BoundBreakpoint RegEx="24,0 to 24,0 in (\[Unknown\]|libbreakpoint.so)!break_test_1()"></BoundBreakpoint>
     </Event>
```

Comment:
is there any way to fix the race condition rather than have a regex in all these tests? As more tests will be added this seems like it will be annoying to maintain as the Regex will never be part of the session log.